src/VirtualBarometer.h

```diff
@@ -0,0 +1,22 @@
+#ifndef VIRTUAL_BAROMETER
+#define VIRTUAL_BAROMETER
+#include "PhoenixPositionProvider.h"
+#include <random>
+#include "Vector3.hpp"
+
+class VirtualBarometer {
+    public:
+        VirtualBarometer(unsigned int rs, double alt, double ns);
+        double get_reported_altitude();
+        void process(double);
+        PhoenixPositionProvider position_provider;
```

Comment:
This should probably be a pointer to `position_provider`. Currently, it is declared as a unique instance that is a member of this class. Consider change to `PhoenixPositionProvider* position_provider`.